Automattic/jetpack
# Move gutenberg-base-styles file to js-packages/base-styles

Fixes #

#### Changes proposed in this Pull Request:
Moved gutenberg-base-styles.scss file to js-packages/base-styles and updated its imports in the Jetpack plugin directory


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Make sure `pnpm build-extensions` runs fine
* Checkout master on your local or proxy site.
* Create a new post and a couple of blocks from this list: business-hours, contact-form, donations, instagram-gallery, slideshow or one of the other components touched in this PR
* Pull this branch and open the saved draft post. Everything should be looking as it was earlier.
